node/service/src/chain_spec.rs

```diff
@@ -117,6 +117,50 @@ pub fn wococo_config() -> Result<PolkadotChainSpec, String> {
 	PolkadotChainSpec::from_json_bytes(&include_bytes!("../res/wococo.json")[..])
 }
 
+/// The default parachains host configuration.
+fn default_parachains_host_configuration() -> polkadot_runtime_parachains::configuration::HostConfiguration<BlockNumber> {
+	polkadot_runtime_parachains::configuration::HostConfiguration {
+		validation_upgrade_frequency: 1u32,
+		validation_upgrade_delay: 1,
+		code_retention_period: 1200,
+		max_code_size: MAX_CODE_SIZE,
+		max_pov_size: MAX_POV_SIZE,
+		max_head_data_size: 32 * 1024,
+		group_rotation_frequency: 20,
+		chain_availability_period: 4,
+		thread_availability_period: 4,
+		max_upward_queue_count: 8,
+		max_upward_queue_size: 1024 * 1024,
+		max_downward_message_size: 1024,
```

Comment:
Would it be ok to change this?
Would be great if we can send sth. like e.g. a `set_storage` with a bunch of keys (e.g. 20).

Reply:
What do we use in Kusama?